pkg/controller/scylladbdatacenter/resource_test.go

```diff
@@ -2712,8 +2712,11 @@ func Test_MakeManagedScyllaDBConfig(t *testing.T) {
 					Namespace: "foo-ns",
 					Name:      "foo-managed-config",
 					Labels: map[string]string{
-						"scylla/cluster": "foo",
-						"user-label":     "user-label-value",
+						"app":                          "scylla",
+						"app.kubernetes.io/managed-by": "scylla-operator",
+						"app.kubernetes.io/name":       "scylla",
+						"scylla/cluster":               "foo",
```

Comment:
the name of the resource and ClusterName is the same in the test, wouldn't it be better to make them different to make sure the right one is propagated to the label?

Reply:
good point, changed